app/recordtransfer/emails.py

```diff
@@ -69,20 +81,31 @@ def send_submission_creation_success(form_data: dict, submission: Submission) ->
 
 
 @django_rq.job
-def send_submission_creation_failure(form_data: dict, user_submitted: User) -> None:
+def send_submission_creation_failure(
+    form_data: dict, user_submitted: User, recipient_emails: Optional[List[str]] = None
+) -> None:
     """Send an email to users who get submission email updates that a user submitted a new
     submission and there WERE errors.
 
     Args:
         form_data (dict): A dictionary of the cleaned form data from the submission form. This is
             NOT the CAAIS tree version of the form.
         user_submitted (User): The user that tried to create the submission.
+        recipient_emails (List[str], optional): A list of recipient email addresses to send the notification to. If not provided, admin recipients will be used.
     """
     subject = "Submission Failed"
-    recipient_emails = _get_admin_recipient_list(subject)
-
+    admin_recipients = _get_admin_recipient_list(subject)
+    if admin_recipients:
+        used_recipients = admin_recipients
+    elif recipient_emails:
+        used_recipients = recipient_emails
+    else:
+        LOGGER.warning(
+            "No admin recipients found for submission creation success email. Skipping send."
+        )
+        return
```

Comment:
Same comment as above.

```suggestion
    used_recipients = recipient_emails or _get_admin_recipient_list(subject)

    if not used_recipients:
        LOGGER.warning(
            "No admin recipients found for submission creation success email. Skipping send."
        )
        return
```